src/snowflake/snowpark/session.py

```diff
@@ -265,6 +265,9 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
+    def __str__(self):
+        return f"<{self.__class__.__module__}.{self.__class__.__name__}: account={self.get_current_account()}, role={self.get_current_role()}, database={self.get_current_database()}, schema={self.get_current_schema()}, warehouse={self.get_current_warehouse()}>"
```

Comment:
Just curious if we could line break this in code? e.g.

```
return (f"<{self.__class__.__module__}.{self.__class__.__name__}: "
        f"account={self.get_current_account()}, "
        ...)

```

Reply:
True, that looks a lot better, I wonder why flake8 doesn't catch this.